### ISSUE TITLE
DISSENY - Qui som

### ISSUE THREAD

**Issue by andriusmp** [iboadella/encercla_client#71]
- Posar títol (Qui som / Quiénes somos)
- Guionets ben identats
- Enllaços clicables
![image](https://user-images.githubusercontent.com/42779858/48011992-46098700-e121-11e8-8fe0-619378de9291.png)

- Han d'haver-hi tres subtítols al mateix nivell: Associació pel Desenvolupament Rural de la Catalunya Central (ADCC), Inèdit Innovació, Economia Circular
- Aquests tres títols marquen tres subseccions. A dintre van els textos del excel de Continguts.

**Comment by iboadella** [comment]
fet

**Comment by andriusmp** [comment]
No, els títols no estan com toca, falta un títol i tal com està identat el text no s'enten:
![image](https://user-images.githubusercontent.com/42779858/48792660-1c418a00-ecf5-11e8-8cf5-b0e339aaac60.png)

**Comment by iboadella** [comment]
fet

**Comment by andriusmp** [comment]
Ok!
Només falta treure aquest espai tant gran entre la descripció d'inèdit i el logo i hi ha cometes que sobren (Al primer paràgraf, després del link del web de catcentral, i al final de l'últim paràgraf.

Quan es fa clic al link de catcentral, s'obre en la mateixa pestanya i fa que l'usuari surti de la sessió. Podem fer que s'obri en un altra pestanya?
